modules/doc/content/help/development/VSCode.md

```diff
@@ -70,6 +70,9 @@ The following settings are recommended to pass the MOOSE GitHub CI pre-checks:
         "${env.CONDA_PREFIX}/**",
         "/usr/include/**"
     ],
+    "files.associations": {
+        "*.C": "cpp",
+    },
     "workbench.editorAssociations": {
         "*.C": "cpp",
         "*.h": "cpp"
```

Comment:
I think those are conceptually wrong. The editor is not a file type, but a UI component that can be fundamentally different from a text editor. E.g. flame graph viewer for profile files.

Reply:
well they rely on it so I dont think we ought to change this here

Reply:
Who relies on it? These settings are invalid:

![image](https://github.com/idaholab/moose/assets/202302/3933a448-44ff-4507-9858-8f51a3320ae7)

```
Value is not accepted. Valid values: "default", "workbench.editorinputs.searchEditorInput", "workbench.editor.chatSession", "jupyter-notebook", "vscode.markdown.preview.editor", "imagePreview.previewEditor", "vscode.audioPreview", "vscode.videoPreview", "jsProfileVisualizer.cpuprofile.table", "jsProfileVisualizer.heapprofile.table", "amiga.profile", "amiga.objdump", "amiga.savestate", "gitlens.rebase", "workbench.editors.gettingStartedInput".(1)
```

Reply:
alright I ll delete them